tests/skyserve/update/old_server.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+import http.server
+import socketserver
+
+
+class MyHttpRequestHandler(http.server.SimpleHTTPRequestHandler):
+
+    def do_GET(self):
+        # Return 200 for all paths
+        # Therefore, readiness_probe will return 200 at path '/health'
+        self.send_response(200)
+        self.send_header('Content-type', 'text/html')
+        self.end_headers()
+        html = """
+            <html>
+            <head>
+                <title>SkyPilot Test Page</title>
+            </head>
+            <body>
+                <h1>Hi, SkyPilot here!</h1>
```

Comment:
```suggestion
                <h1>Hi, old SkyPilot here!</h1>
```

Reply:
The original text might be fine, as long as ew can distinguish it with the new text.